make/config/kind/v1beta2.yaml

```diff
@@ -13,5 +13,9 @@ kubeadmConfigPatches:
       name: config
     networking:
       serviceSubnet: 10.0.0.0/16
+    etcd:
+      local:
+        extraArgs:
+          unsafe-no-fsync: "True"
```

Comment:
background: https://github.com/etcd-io/etcd/pull/11946
```
This makes it possible to run an etcd node for testing and development
without placing lots of load on the file system.
```

fsync syscall: https://man7.org/linux/man-pages/man2/fdatasync.2.html
```
fsync() transfers ("flushes") all modified in-core data of (i.e.,
       modified buffer cache pages for) the file referred to by the file
       descriptor fd to the disk device (or other permanent storage
       device) so that all changed information can be retrieved even if
       the system crashes or is rebooted.  This includes writing through
       or flushing a disk cache if present.  The call blocks until the
       device reports that the transfer has completed.
```

Setting this flag will omit fsync syscalls.
The flag is unsafe because in case of a crash of an etcd node, there might
be data loss. Thus, the flag should not be used in production! For testing
however, losing the etcd state in case of a node crash is not a problem at
all and skipping the fsync syscalls reduces the diskload.

Reply:
Thanks for the context! :+1: 

Would you agree that it's worth adding a comment to this YAML file explaining why we have this argument set, and why it's safe in a testing environment?

(As a security person if I see "unsafe" I think "some security check is not being done here" and would be suspicious)

Reply:
@SgtCoDFish I have added a comment explaining why it isn't unsafe for us: c598132d9